bin/optimize-schemas.ts

```diff
@@ -90,6 +90,10 @@ forEachJsonFile(pathToSchemas, (pathToSchema) => {
           ) {
             value.additionalProperties ||= false;
           }
+
+          if (key === "url" || (key.endsWith("_url") && key !== "ssh_url")) {
```

Comment:
`ssh_url`, e.g. `git@github.com:Octocoders/Hello-World.git`, doesn't match the uri format https://github.com/octokit/webhooks/blob/46983cd8739c706639c066eaed538c768c4d9cc3/payload-examples/api.github.com/repository/created.payload.json#L73